index.html

```diff
@@ -141,80 +141,3 @@ <h2>$1200</h2>
 </body>
 </html>
```

Comment:
- required

- Well done so far👏👏

- [ ] Please explain the why you are asking for that change by providing examples and best practices approach ✔️ so the student can learn something 👨‍🏫😄 as well and don't feel the change was unfair 👌.
- [ ]  I suggest you add emoji's to your comments so that the student understands the tone you are trying to communicate 😎.